pombola/templates/default_base.html

```diff
@@ -9,6 +9,8 @@
 {% endcomment %}
 {% block ga_experiment %}
 {% endblock %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
move to the rest of the CSS link tags

Reply:
Done. Also, I discovered the ordering matters. If I move it any lower, it alters the page CSS significantly.